src/backend/src/prisma-query-args/auth-user.query-args.ts

```diff
@@ -1,9 +1,11 @@
 import { Prisma } from '@prisma/client';
+import projectQueryArgs from './projects.query-args';
 
 const authUserQueryArgs = Prisma.validator<Prisma.UserArgs>()({
   include: {
     userSettings: true,
-    teamAsLead: true
+    teamAsLead: true,
+    favoriteProjects: {...projectQueryArgs}
```

Comment:
lets just make this include true, and then well just do favoritedProjectsId like we do with teamaslead